maquetado/scss/styles.scss

```diff
@@ -157,3 +191,54 @@
     left: 0px;
   }
 }
+
+.login-img {
+  height: 70px;
+  margin: 0 24px;
+  width: 210px;
+}
+
+.login-detail {
+  background-color: $earls-green;
+  border-radius: 5px;
+  height: 30px;
+  width: 250px;
+  padding: 0 90px;
+  position: relative;
+  &::before {
+    content: '';
+    border-bottom: 2px solid $gray;
+    bottom: 0;
+    left: 0px;
+    margin-bottom: -20px;
```

Comment:
you shouldn't use a margin in an element with position absolute. Move it using `top, bottom, right, left`